temporal-spring-boot-autoconfigure-alpha/README.md

```diff
@@ -57,7 +57,13 @@ spring.temporal:
       # insecure-trust-manager: true # or add ca.pem to java default truststore
 ```
 
-to set the key and cert chain directly from the environment variables for example.
+## Data Converter
+
+Define a bean of type `io.temporal.common.converter.DataConverter` in Spring context to be used as a custom data converter.
+If Spring context has several beans of the `DataConverter` type, they will be ignored except the one named `mainDataConverter`.
```

Comment:
Why do we allow other data converters that are just gonna be ignored? (forgive my Spring ignorance here)

Reply:
Because the Spring context is not just ours and not just for us. It's the context of the user app. And what people like to do (especially with XML-based configs) is to make everything beans.
Now imagine a user configures a custom data converter with codecs. This will lead a user to create a DefaultDataConverter and a CodecDataConverter, which are both DataConverter. And a very common way is to make all of these thingys beans even if they are directly referenced in only one place.